src/components/page-step/index.tsx

```diff
@@ -221,32 +214,11 @@ export class PageStep extends Component<PageStepsProps, { now: Date }> {
                 </Typography>
               ))}
             <section className="step-log">
-              <Typography
-                variant="h4"
-                className={`step-log-header${
-                  this.props.stepLog ? '-absolute' : ''
-                }`}
-              >
-                Log
-              </Typography>
-              <AsyncResource
-                resource="JOB_LOGS"
-                resourceParams={[appName, jobName]}
-              >
-                {this.props.stepLog ? (
-                  <Code
-                    copy
-                    download
-                    filename={`${appName}_${jobName}`}
-                    autoscroll
-                    resizable
-                  >
-                    {this.props.stepLog.replace(/\r/gi, '\n')}
-                  </Code>
-                ) : (
-                  <Typography>No logs</Typography>
-                )}
-              </AsyncResource>
+              <JobStepLogs
+                appName={appName}
+                jobName={jobName}
+                stepName={stepName}
+              ></JobStepLogs>
```

Comment:
Elements without children can be shortened

```suggestion
              <JobStepLogs
                appName={appName}
                jobName={jobName}
                stepName={stepName}
              />
```